.github/workflows/django-ory-cloud.yml

```diff
@@ -0,0 +1,31 @@
+name: E2E Tests for django-ory-cloud
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+jobs:
+  test-playwright:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./django-ory-cloud
```

Comment:
This is awesome, I didn't know one can set defaults like this!

On the other hand, you only need to set the working directory for the "Run playwright tests" step. Set the working directory only for that step so that it doesn't interfere with the other steps?

Reply:
Ah yea makes sense, for some reason i thought it needs to run everything in the folder. But with only run playwright it works fine, so not really a change here 😅